integration_testing/features/cancel-a-sync-operation.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Cancel a sync operation

```

Comment:
Let's simplify filenames and feature names by rephrasing to eliminate indeterminate articles (e.g. `cancel-sync` here, `cancel-copy` below, etc.)